Client/Frontend/Browser/Tab Management/Tab.swift

```diff
@@ -527,8 +523,6 @@ class Tab: NSObject {
 
         url = currentlyOpenUrl
         sessionData = SessionData(currentPage: 0, urls: [currentlyOpenUrl], lastUsedTime: Date.now())
-        historyList = [currentlyOpenUrl]
-        webView = nil
```

Comment:
Yes, this seems odd. But, you'll have noticed that the `set` for `historyList` wasn't doing anything, so the first line of code is.... doing nothing. And the second line, where `webview` is set to `nil` you'll notice, if you expand the file in GH to see what the `close` function does.... well... it's already there. But the `close` function basically did nothing, because we'd originally set the `webview` to nil???? So weird. All fixed now.